backend/ContextAlgorithm/ContextAlgorithm.py

```diff
@@ -0,0 +1,133 @@
+import requests
+import xml.etree.cElementTree as ET
+import string
+
+INPUT_LANG = 'es'
+OUTPUT_LANG = 'en'
+WORD = 'cama'
+WORD_DIFFICULTY = 882
```

Comment:
This is because I don't have the input yet. That number is the position of "cama" in the frecuency list. When I'll define the input, WORD, WORD_DIFFICULTY and INPUT_LANG will have the value of the input values